subprojects/tooling-api/src/testFixtures/groovy/org/gradle/integtests/tooling/fixture/ToolingApiSpecification.groovy

```diff
@@ -242,4 +242,12 @@ abstract class ToolingApiSpecification extends Specification {
     protected static GradleVersion getTargetVersion() {
         GradleVersion.version(targetDist.version.baseVersion.version)
     }
+
+    /**
+     * @return the current <b>base</b> version of Gradle being used to run this test
+     */
+    protected static GradleVersion getCurrentVersion() {
+        GradleVersion.current().baseVersion
+    }
+
```

Comment:
> >This will make it inconsistent with how we handle targetVersion.

>I don't understand what you mean by that. Can you explain?

There are already a number of places where we check the target version of Gradle being tested (as opposed to the current version)  
```groovy
 if (targetVersion >= GradleVersion.version("3.4")) {
```
So I think the checks should be consistent.

As for readability of code:
I think this:
```groovy
if (currentVersion < GradleVersion.version('4.0')) {}
if (currentVersion >= GradleVersion.version('3.1')) {}
```
is easier to read than this
```groovy
if (currentVersionSmallerThan(GradleVersion.version('4.0'))) {}
if (currentVersionGreaterThanOrEqualTo(GradleVersion.version('3.0'))) {}
```